.github/actions/cli-rust-setup/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: "CLI Rust Setup"
+description: "Setup the rust toolchain and cache for the CLI build"
+inputs:
+  GIT_CREDENTIALS:
+    description: "Optional credentials to pass to git"
+    required: false
+  ADDITIONAL_KEY:
+    description: "An optional additional key to pass to rust-cache"
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - uses: dsherret/rust-toolchain-file@v1
+    # rust-cache action will cache ~/.cargo and ./target
+    # https://github.com/Swatinem/rust-cache#cache-details
+    - name: Run cargo cache
+      uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84 # pin@v2.7.3
```

Comment:
Please update to v2.7.8 as seen in https://github.com/aptos-labs/aptos-core/commit/a8790a21e96c8fb53179b212afcabd27100c563e